benchmark/jsonmask-java-benchmark/README.md

```diff
@@ -0,0 +1,32 @@
+# Overview
+This subproject contains performance measurement for [json-mask-java]() library.
```

Comment:
If this is a separate project, you should take it out from the main project to one level up. Otherwise it looks like a sub-module and confuses a bit